<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MipController</class>
 <widget class="QMainWindow" name="MipController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MipController</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="BTTabSystem">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="setupTab">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QPushButton" name="startBTScanButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Scan</string>
      </property>
     </widget>
     <widget class="QListWidget" name="deviceListWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>531</width>
        <height>192</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="logTab">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QTextEdit" name="statusLog">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>661</width>
        <height>341</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="controlTab">
     <attribute name="title">
      <string>Controller</string>
     </attribute>
     <widget class="QWidget" name="commsPanel" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>631</width>
        <height>331</height>
       </rect>
      </property>
      <widget class="QPushButton" name="sendHexButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>221</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Send Hex</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="commsLog">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>331</width>
         <height>301</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>10</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="hexCommandInput">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
